### PR TITLE
MGXS examples pin power plots normalization

### PR DESCRIPTION
**Pull request by GiudGiud**
Hello, first pull request here

The guide tubes were making the normalization of the pin powers wrong in the MGXS examples. 

Issues I still don't know how to deal with:
- the images (not the matplotlib plots) have a really really long file name. Therefore all the text editors I have (tried 3) cut the line with the file name in 2-3 parts. Is that normal? Where is the picture hosted?
- is there anything else I need to update when changing an iPython notebook?
- @johnnyliu27 is updating the same files in a simultaneous pull request. What's the rule? Whoever gets through first doesn't need to merge the other's stuff?

@smharper

Thanks!

### CODE REVIEW

File: examples/jupyter/mg-mode-part-i.ipynb [hunk 1]
```diff
@@ -404,7 +404,8 @@
...
...7pvXPoS7UpEdkgIj8s+rpDRGZVzl1T2PG4iHwgsx1L\nReQxEXlVRCaMc13acXHRz34RuTpnX5a+t4jIsyKyt9I2W0TuEZEniu0JmW04XUTuE5F9xe/jr0Zk\nxxtE5Lsi8v3Cjr8v2s8SkYcKO24VkaODlatqr38AHF/Z/0sANxT7lwDYiUF6+HsBPJTZjj8FcFSx\nfx2A64r9cwB8H8AxAM4C8GMAMzLacTaAdwC4H8BEpb0zOwDMKPS/FcDRRb/ndPg3cT6A9wDYW2n7\nRwBXF/tXl7+fjDbMAfCeYv9NAH5U/A66tkMAHFfszwTwUHE/3AZgsmi/AcBfhOru/ZOD9qTalKp+\nS1VfKQ4fxCANvbRjm6q+rKo/xSDZ7NyMduxT1cctp7q041wA+1X1J6r6GwDbiv47QVUfAPC80bwY\nwNZifyuAD2W24RlVfaTY/yWAfRgUNOraDlXVXxWHM4sfBfB+AF9rY0fvnQMwqDYlIk8B+HO8Vqey\nrtpUFyzD4Kll1HZU6dKOvnzmKqdqURag2J7SVcciciaAd2PwX7tzO0Rkhog8CuBZAPdg8FR3uPLP\nLOr30wvnkLvaVCo7imvWAHilsGVkdtjEUtvRk756jYgcB2A7gL82nnI7Q1V/p4Miz3MxeKo723ZZ\nqN7WNSRToJmrTaWyQ0SuAPBBABdoMZgbhR01JLejJ335clBE5qjqM8Xw8tncHYrITAwcwy2q+vVR\n2VGiqodF5H4MYg6zROSo4ukh6vfTiyeHJvpSbUpELgawGsClqvpi5dSdACZF5BgROQuDcvzfzWVH\nA13a8T0A84qI+NEAJov+R8mdAK4o9q8AsCNnZyIiAG4CsE9VvzBCO04u35yJyLEALsQg/nEfgI+0\nsiNnJDVRNHY7gL0A9gD4dwCnVaK0mzAYX/0PKpH7THbsx2Cc/Wjxc0Pl3JrCjscBLMpsx4cx+M/9\nMoCDAO5j8QNEAAAAfklEQVQekR2XYBCh/zGANR3/TXwVg0rmvy2+i08BOBHAvQCeKLazM9vwxxg8\nqu+p/E1cMgI7FgD478KOvQDWFu1vxeCfw34AtwM4JlQ3Z0gSQqz0flhBCBkNdA6EECt0DoQQK3QO\nhBArdA6EECt0DoQQK3QOhBArdA6EECv/D8nj1UuvNoxlAAAAAElFTkSuQmCC\n",
```
Comment (paulromano) [inline examples/jupyter/mg-mode-part-i.ipynb]: For some reason, your revised notebooks have linebreaks in the middle of the "image/png" data that is preventing the documentation from building successfully.

**Comment by smharper** [comment]
Thanks for the PR, @GiudGiud!

To answer both your points 1 and 2, don't use a text editor to modify .ipynb notebooks.  Use Jupyter.  (I believe you can install it with pip using a command like `pip3 install jupyter`.)  Whenever you're done modifying the notebook, restart the kernel (under the "Kernel" tab at the top of the screen), and run all the cells ("Cell" tab then, "Run All").  That will ensure that all the output cells in the notebook are consistent with the inputs.

You're right on point 3.  Whoever's gets merged second has the responsibility of merging in the other one.

**Comment by GiudGiud** [comment]
Thanks for reviewing @paulromano @smharper !
The line breaks were inserted by my text editor. I don't have jupyter and I am making very minor changes so I didn't see that coming.
I won't modify the notebooks without Jupyter again though!

**Comment by paulromano** [comment]
Thanks @GiudGiud, and congrats on your first PR!

**Pull request merged**